examples/tfengine/org_foundation.hcl

```diff
@@ -84,3 +84,23 @@ template "org_policies" {
     ]
   }
 }
+
+# Top level prod folder.
+template "folder_prod" {
+  recipe_path = "{{$recipes}}/folder.hcl"
+  output_path = "./live/prod"
+  data = {
+    display_name = "prod"
+  }
+}
+
+# Prod folder for team 1.
+template "folder_team1" {
+  recipe_path = "{{$recipes}}/folder.hcl"
+  output_path = "./live/prod/team1"
+  data = {
+    parent_type                  = "folder"
```

Comment:
Do you need a parent_id here?

Reply:
The add_parent_folder_dependency below this field should take care of that. Any suggestions on making that more obvious?

Reply:
I see. I'm not really sure how to make it more clear, let's leave it for now.